packages/@okta/vuepress-site/docs/guides/device-context/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Device Context
```

Comment:
@chrisdavidmills - your thoughts on Dan's comments?

Reply:
This would really require a conversation with Jeff; we'd need to figure out the best way to phrase the title so that it is accurate, but also presents the user with a meaningful use case choice. "Device Context" is pretty meaningless, but then again so is  "AMFA"

Reply:
@robertnicolo-okta ^